### PR TITLE
Export total metrics in system module

### PR DESCRIPTION
**Pull request by monicasarbu**

### CODE REVIEW

**Comment by monicasarbu** [comment]
Here is the result. A Kibana dashboard for Network statistics. 

metricbeat network dashboard

File: metricbeat/metricbeat.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ metricbeat.modules:
 - module: system
   metricsets:
     # CPU stats
-    - cpu
+    #- cpu
```
Comment (andrewkroh) [inline metricbeat/metricbeat.yml]: Did you intend to check in these changes?

**Comment by andrewkroh** [comment]
There's a test failure. `Exception: Key 'system.network.total.bytes' found in event is not documented!`

**Comment by andrewkroh** [comment]
Could we use a scripted field in Kibana to derive the totals instead of storing them in ES?

**Comment by ruflin** [comment]
👍  on the proposal of @andrewkroh It would be interesting to find out how "complex" it is to do rather simple stuff (like total) directly in ES instead of sending the data.

**Comment by monicasarbu** [comment]
Yes, I agree it would be great to use the scripted fields. I will give it a try and see how it works.

**Comment by monicasarbu** [comment]
Close it as the total values are done by scripted fields in Kibana

**Pull request closed without merging**